stdlib/3.4/asyncio/locks.pyi

```diff
@@ -17,8 +19,8 @@ class _ContextManagerMixin(Future[_ContextManager]):
     # Apparently this exists to *prohibit* use as a context manager.
     def __enter__(self) -> object: ...
     def __exit__(self, *args: Any) -> None: ...
-    def __aenter__(self): ...
-    def __aexit__(self, exc_type, exc, tb): ...
+    def __aenter__(self) -> Awaitable[object]: ...
```

Comment:
I might need to think more about whether and how it makes a difference, but we've usually annotated coroutine methods with `@types.coroutine` instead of using Awaitable as the return type.

Also, we should avoid putting the `object` there. I haven't looked at the code yet, but maybe we can use a self type?